projects/web/src/components/submit.tsx

```diff
@@ -25,13 +19,14 @@ export function Submit(props: SubmitProps) {
         'Content-Type': 'application/json',
         'Authorization': 'Bearer ' + props.authState.getToken().idToken,
       },
+      credentials: 'include',
```

Comment:
I see this has snuck back in ;)

> The credentials option specifies whether fetch should send cookies and HTTP-Authorization headers with the request.

Because we are explicitly adding the bearer token in the Authorization header, we don't need to set it.